src/components/settings/timelapse/TimelapseSettings.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <div>
+    <v-subheader id="timelapse">
+      {{ $t('app.timelapse.title.timelapse') }}
+    </v-subheader>
+    <v-card
+      :elevation="5"
+      dense
+      class="mb-4"
+    >
+      <app-setting
+        :title="$tc('app.general.title.camera', 1)"
```

Comment:
It seems the existing settings labels are under `app.setting.label.*`, so maybe move all of these there for consistency reasons?

Reply:
I'm not a big fan of duplicating translations, though.. Seems like `app.general.title.camera` is used everywhere else, and it fits here perfectly, so I don't see a need to create a new label for it.

Reply:
Ah, I may have misunderstood - do you want me to move the `app.timelapse.setting.*` keys to `app.setting.label.*`?

Reply:
Correct, on this case I'm just saying to move the parent path of these translation keys!

Reply:
Honestly not sure about this, as this is an entirely new "component" and dynamically enabled when the timelapse component is supported in moonraker (bedmesh has its own section too). I think it would clutter the settings labels subkey too much.. What do you think?

Reply:
Fair enough, let's leave it be!